roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -99,13 +99,32 @@
     - "{{ ceph_keys.get('stdout_lines') | default([]) }}"
   when: cephx
 
+- name: check if libvirt keyring exists
+  command: find /etc/ceph -name "{{ cluster }}.client.libvirt.keyring"
+  changed_when: false
+  register: is_libvirt_keyring_exists
+  always_run: true
+  run_once: true
+  when: cephx
+
+- name: set ceph libvirt keyring permission
+  shell: chown ceph:ceph *.libvirt.keyring ; chmod 0640 *.libvirt.keyring
```

Comment:
@leseb In task `set keys permissions` of this file `roles/ceph-mon/tasks/ceph_keys.yml`  owner/group/mode are not configurable with variables as well. They are simply used the way they are used in a `file` module. That's what I had done in my first commit for the task `set ceph libvirt keyring permission` that I had added.

Do owner/group/mode actually need to be configurable with variables?